h/api/schemas.py

```diff
@@ -256,7 +256,7 @@ def validate(self, data):
         # group they've asked to create one in.
         if 'group' in appstruct:
             if appstruct['group'] == '__world__':
-                group_principal = security.Everyone
+                group_principal = security.Authenticated
```

Comment:
I remember I raised this in a PR. This shouldn't have any side-effects since unauthenticated users couldn't even hit this code path. :+1: